config.json

```diff
@@ -222,6 +222,19 @@
         ],
         "status": "beta"
       },
+      {
+        "name": "Sorting Room",
+        "slug": "sorting-room",
+        "uuid": "411f4f1e-7ee2-4f11-955d-8af86da69c2a",
+        "concepts": [
+          "type-assertion",
+          "type-conversion"
```

Comment:
FYI: It does check, but it ignores exercises marked "wip" so that you can stage changes like this:

> The "exercises.concept[].concepts" values must not be in any other concept exercise's "concepts" property
> https://github.com/exercism/configlet/issues/249

Reply:
Thanks for clarifying!